### ISSUE TITLE
CRED: Examine namespace to be mapped to the "title_id" field in the KBART export 

### ISSUE THREAD

**Issue by kristenwilson** [k-int/gokb-phase1#304]
The KBART export function by default populates the title_id field with the GOKb title ID. However, if we want to support publishers as a place to house their KBART files, they need to be able to produce an export that contains their own title IDs in this field (if available). One possible solution may be to set a preferred namespace for the package. Another may be to allow the user to select a namespace manually. This needs a little discussion.

**Comment by ianibo** [comment]
Yeah.. My preference [not deeply considered] is a parameter that lets the
user specify at export time which namespace is to be put into title_id.

eg... http://..../?...&title_id_ns=MyNS

On 11 December 2014 at 17:58, kristenwilson notifications@github.com
wrote:

> The KBART export function by default populates the title_id field with the
> GOKb title ID. However, if we want to support publishers as a place to
> house their KBART files, they need to be able to produce an export that
> contains their own title IDs in this field (if available). One possible
> solution may be to set a preferred namespace for the package. Another may
> be to allow the user to select a namespace manually. This needs a little
> discussion.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/k-int/gokb-phase1/issues/304.

## 

Ian Ibbotson
W: http://ianibbo.me
E: ianibbo@gmail.com
skype: ianibbo
twitter: ianibbo
GoodReads: https://www.goodreads.com/user/show/3106516-ian-ibbotson
